v1/ratelimit/redis_delayed_sync.go

```diff
@@ -209,16 +209,6 @@ func (r *RedisDelayedSync) sync(key string, expiry int64) error {
 		}
 	}
 	if !hasSyncedBefore {
-		// Case: The key is set by another server and the current server joins the cluster later after cluster inactivity but before key expiry
-		// We need to reset the key in redis to the local resetAt value
-		if remoteValue < resetAt {
-			cmd := r.redisClient.Set(r.ctx, key, resetAt, 0)
-			if cmd.Err() != nil {
-				return cmd.Err()
-			}
-			r.lastSyncedResetAt.Store(key, resetAt)
-			return nil
-		}
```

Comment:
This is unintended as it raises reset_at for other servers when new servers join. If the other servers, have been inactive, it would caused a large increase